allure-rest-assured/src/main/java/io/qameta/allure/restassured/AllureLoggerFilter.java

```diff
@@ -0,0 +1,68 @@
+package io.qameta.allure.restassured;
+
+import io.qameta.allure.curl.CurlBuilder;
+import io.qameta.allure.curl.FreemarkerUtils;
+import io.restassured.filter.Filter;
+import io.restassured.filter.FilterContext;
+import io.restassured.internal.NameAndValue;
+import io.restassured.response.Response;
+import io.restassured.specification.FilterableRequestSpecification;
+import io.restassured.specification.FilterableResponseSpecification;
+import ru.yandex.qatools.allure.Allure;
+import ru.yandex.qatools.allure.events.MakeAttachmentEvent;
+import ru.yandex.qatools.allure.events.StepFinishedEvent;
+import ru.yandex.qatools.allure.events.StepStartedEvent;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AllureLoggerFilter implements Filter {
+
+    public static AllureLoggerFilter log() {
+        return new AllureLoggerFilter();
+    }
+
+    @Override
+    public Response filter(FilterableRequestSpecification requestSpec, FilterableResponseSpecification responseSpec,
+                           FilterContext filterContext) {
+
+        AllureData allureData = new AllureData();
+        Response response = filterContext.next(requestSpec, responseSpec);
+
+        String uri = requestSpec.getURI();
+        String method = requestSpec.getMethod();
+        String requestBody = requestSpec.getBody();
+
+        allureData.setRequestUrl(uri);
+        allureData.setRequestMethod(method);
+        allureData.setRequestBody(requestBody);
```

Comment:
> @viclovsky how about showing both request and response at the same time in two vertical tabs, when log screen is expanded?

I'd prefer to keep it as is. Nevertherless, I'll add customization for view. Will it work for you?

>  what about handling multiparts? Didn't found any related code...

It is not supported yet. I'll take a look on it.

> According to screens body is not prettified

Will do it